infoblox_client/connector.py

```diff
@@ -485,13 +488,24 @@ def delete_object(self, ref, delete_arguments=None):
         return self._parse_reply(r)
 
     @staticmethod
-    def is_cloud_wapi(wapi_version):
```

Comment:
all i am worried about that is if someone is directly calling this function in any other code, the Ansible plugin or openstack plugin, it might break this, i would suggest we keep the naming convention as it is , you can keep the comments so that it makes sense. Other than this everything else looks good

Reply:
Alright, I will change it back.